include/gmock-win32.h

```diff
@@ -55,8 +55,16 @@ struct mock_module_##func \
 #define MOCK_MODULE_FUNC0_CDECL_CONV(m, ...) MOCK_MODULE_FUNC0_CALLCONV(__cdecl, m, __VA_ARGS__)
 
 #define MOCK_MODULE_FUNC1_(tn, constness, ct, func, ...) \
+_Pragma( "optimize( \"\", off )" ) \
 struct mock_module_##func \
 { \
+    static void hideCodeFromOptimizer() { \
```

Comment:
Could we move this code to a separate global function  (with a disabled optimization)? This would help us avoid duplicating this code in each `struct mock_module_##func` version.

Reply:
Let me try...